sound/soc/sof/control.c

```diff
@@ -471,11 +471,11 @@ int snd_sof_bytes_ext_put(struct snd_kcontrol *kcontrol,
 	}
 
 	/* notify DSP of byte control updates */
-	snd_sof_ipc_set_get_comp_data(sdev->ipc, scontrol,
-				      SOF_IPC_COMP_SET_DATA,
-				      SOF_CTRL_TYPE_DATA_SET,
-				      scontrol->cmd,
-				      true);
+	ret = snd_sof_ipc_set_get_comp_data(sdev->ipc, scontrol,
+					    SOF_IPC_COMP_SET_DATA,
+					    SOF_CTRL_TYPE_DATA_SET,
+					    scontrol->cmd,
+					    true);
```

Comment:
@juimonen @singalsu I think this fixes an error rather than change the functionality. I don't think we ever expected to return the pm status here?
@ranj063 wadyathink?

Reply:
also the commit message should be ASOC: SOF: IPC: fix bytes_ext_put() return status

Reply:
@plbossart yes this was an oversight earlier

Reply:
The commit impacts several functions so I kept the commit title more generic. Is it OK?